tools/niminst/buildsh.nimf

```diff
@@ -125,11 +125,16 @@ case $uos in
     ;;
   *darwin* )
     myos="macosx"
-    CC="clang"
```

Comment:
Likewise!

Reply:
@Araq I assume that for MacOS hardcoding Clang does no good, since for x86 and aarch64 Clang is the default compiler anyway (both in Xcode and Macports, so nothing changes), while for ppc Clang should never be invoked (hardcoding breaks the build).
But if there is a reason to hardcode it, then it must exclude ppc and ppc64.